tests/simulator/test_initialization.py

```diff
@@ -656,6 +666,60 @@ def test_levelghostparticles_have_correct_split_from_coarser_particle(self, refi
             self._test_levelghostparticles_have_correct_split_from_coarser_particle(dim, interp_order, refinement_boxes)
 
 
+    def ddt_test_id(self):
+        return self._testMethodName.split("_")[-1]
+
+    def _test_patch_ghost_on_refined_level_case(self, has_patch_ghost, **kwargs):
+        import pyphare.pharein as ph
+        from pybindlibs import cpp
+        from pyphare.simulator.simulator import startMPI
+
+        startMPI()
+
+        out = "phare_outputs"
+
+        test_id = self.ddt_test_id()
+        if cpp.mpi_size() > 1: # SKIP
```

Comment:
could you comment on why this test is skipped for parallel jobs?

Reply:
yes, haven't spent much time on figuring it out tbh